### ISSUE TITLE
includes/admin_blocks

### ISSUE THREAD

**Issue by IntegraMOD** [IntegraMOD/IntegraMOD151#66]
quite a few cases of ereg_replace. I thought I had fixed every one at one time but it looks like the entire file was missed or replaced with an old version

**Comment by vendethiel** [comment]
Are you sure you have the latest GIT version of this file? I don’t see any

**Comment by IntegraMOD** [comment]
Its possible that i overwrote the current file with the IMPortal 130 file. I was working on a responsive swipe template with just phpBB2.0.24, IMP and prosilver

**Comment by vendethiel** [comment]
If you can send me the file I’ll take a look.

**Comment by IntegraMOD** [comment]
Ive since updated it

**Comment by vendethiel** [comment]
Perfecr, just wanted to make sure we’re not missing anything that comes with IMPortal 1.3

**Comment by IntegraMOD** [comment]
I did notice an omission in the 130 install

in all templates/overall_header.tpl
FIND 
```PHP
{META}
```
AFTER ADD
```PHP
{NAV_LINKS}
```

without it sid is not passed by any portal links and using them will log you out

**Comment by vendethiel** [comment]
I'll close this one. I think @MWE001 has a separate repository for IM Portal.

**Comment by MWE001** [comment]
Well, I do but have not done anything at all with it. I truly am not a coder as we all know. I am a tester. I get s*it lucky sometimes and make things work. When I went to school, they taught us all old school stuff that was well outdated by the time I graduated. High school kids knew much more than I did with a college degree. So I gave up.  

I figured once Integramod was doing it's thing again if it ever did, then I could tear into IM Portal and keep it active and work on it here and there if needs be or to ever exercise my brains like I was originally doing with it. I just grabbed all files I could in case integramod.com ever went away and the git account was gone. :-)
